db/post_migrate/20220201162748_rename_thumbsup_reactions.rb

```diff
@@ -19,10 +19,31 @@ def up
       SQL
     end
 
-    DB.exec(<<~SQL, alias: alias_name, new_value: original_name)
+    has_both_reactions = DB.query_single(<<~SQL, alias: alias_name, new_value: original_name)
+      SELECT post_id 
+      FROM discourse_reactions_reactions 
+      WHERE reaction_value IN (:alias, :new_value) 
+      GROUP BY post_id 
+      HAVING COUNT(post_id) > 1
+    SQL
+
+    if has_both_reactions
```

Comment:
Doesn't this need to check a rowcount or similar? query_single will return an empty array `[]` which would be truthy?

Reply:
Yes, good catch. Checking for `#present?` should be enough.